include/worker.hpp

```diff
@@ -445,6 +445,14 @@ class Worker
     void processEnabledProperty(const std::string& i_inventoryObjPath,
                                 types::InterfaceMap& io_interfaces);
 
+    /**
+     * @brief API to form asset tag value for the system.
+     *
+     * @param[in] i_parsedVpdMap - Parsed VPD map.
+     * @return - Formed asset tag string. Empty in case of any error.
+     */
+    std::string getAssetTag(const types::VPDMapVariant& i_parsedVpdMap);
```

Comment:
Could have the function named better as  createAssetTagString

Reply:
fixed.

Reply:
agreed